.flake8

```diff
@@ -68,6 +68,7 @@ per-file-ignores =
 copyright-check = True
 copyright-regexp = # Copyright [\d-]+ .*
 copyright-min-file-size = 110
+select = E,F,W,C
```

Comment:
@toofar was this an accidental change in c9aadef48841c8cc8b619064102f9bfee6b82d30?

Reply:
uh, yup. Did `commit -a` after trying something that didn't work